src/Land/Model/SoilWaterParameterizations.jl

```diff
@@ -343,7 +343,7 @@ end
 """
     impedance_factor(
         imp::NoImpedance{FT},
-        θ_ice::FT,
+        θ_i::FT,
```

Comment:
I prefer `θ_ice`, it's more clear and better follows our naming conventions

Reply:
In the Design doc, we use θ_i though... but we can always update the Design doc. What do you think?

Reply:
Where can I go read about the naming conventions?

Reply:
I see. I think the same is true for the atmosphere (the design docs are a bit more compact than the code). I guess we do a bit of mapping in the atmos docs so that we can write things compactly there. IIRC, we used to have a doc called "naming conventions", but maybe it got renamed 😄, the closest thing I found from looking just now is [here](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/VariableList/). Similar variable names exist in the thermodynamics and microphysics modules.

Reply:
We are having a discussion about this, and it seems like _i is acceptable to denote ice (e.g. in Clima parameters, it is used that way for specific heat quantities, with _l for liquid.). It may be that this was not a good practice, and so we'd be propagating it here? But it wasn't difficult for Anna to change it from _ice to _i, so we can change it back if necessary.